test/base/testing_brave_browser_process.h

```diff
@@ -70,9 +70,6 @@ class TestingBraveBrowserProcess : public BraveBrowserProcess {
   tor::BraveTorClientUpdater* tor_client_updater() override;
   tor::BraveTorPluggableTransportUpdater* tor_pluggable_transport_updater()
       override;
-#endif
-#if BUILDFLAG(ENABLE_IPFS)
-  ipfs::BraveIpfsClientUpdater* ipfs_client_updater() override;
```

Comment:
sorry if this is a duplicate question as i haven't checked the entire PR: will we cleanup this component's files (i.e. binary etc)? also the ipfs cache?

Reply:
yes, here the place: https://github.com/brave/brave-core/blob/37735-consider-deprecating-local-node-support-and-ipfs-scheme/components/ipfs/ipfs_component_cleaner.cc#L53-L62